### DISCUSSION TITLE
Importing module pnp.powershell to PowerShell 7.2.11

### DISCUSSION THREAD

**Discussion by MantasSal** [pnp/powershell#3075] [Q&A]
Hello,

I have PowerShell 7.2.11

Can not import module:

import-module pnp.powershell
Import-Module: Could not load file or assembly 'Microsoft.Online.SharePoint.Client.Tenant, Version=16.1.0.0, Culture=neutral, PublicKeyToken=71e9bce111e9429c'. Could not find or load a specific file. (0x80131621)

Could anyone advice?

Thanks

**Comment by AndersRask** [comment]
The issue with colliding assemblies should now be fixed!
https://github.com/pnp/powershell/issues/3207
